app/src/main/java/com/stevesoltys/seedvault/ui/storage/StorageRootResolver.kt

```diff
@@ -57,6 +58,21 @@ internal object StorageRootResolver {
         return roots
     }
 
+    fun getFakeStorageRootForUri(context: Context, uri: Uri): SafOption {
+        return SafOption(
+            authority = AUTHORITY_STORAGE,
+            rootId = "fake",
+            documentId = "fake",
+            // TODO: Use something other than the USB icon?
+            icon = getIcon(context, AUTHORITY_STORAGE, "usb", 0),
```

Comment:
Yeah a different icon would be better. But is this even shown to the user somewhere?